app/src/main/java/org/xwiki/android/sync/activities/SignInViewFlipper.kt

```diff
@@ -316,7 +316,7 @@ class SignInViewFlipper(
             }
 
             override fun onFailure(call: Call, e: IOException) {
-                Log.e(TAG, e.message)
+                Log.e(TAG, "No response from the server.")
```

Comment:
Why do you sure that here may happen only absence of server response?

Reply:
I'll put a more relevant message.